.github/workflows/ci.yml

```diff
@@ -188,17 +188,14 @@ jobs:
       - uses: ./.github/actions/setup
         with:
           cache: restore
-      - name: Maven Package
+      - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $MAVEN clean package ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
+          $MAVEN clean install ${MAVEN_FAST_INSTALL} ${MAVEN_GIB} -pl '!:trino-docs,!:trino-server,!:trino-server-rpm'
```

Comment:
intuitively, this looks redundant. error-prone can compile on its own (this is how i run it locally)

please add a code comment